ee_plugin/utils.py

```diff
@@ -224,15 +276,19 @@ def update_ee_vector_layer(eeObject, layer, shown, opacity):
     return new_layer
 
 
-def add_ee_catalog_image(name, asset_name, vis_params):
+def add_ee_catalog_image(
+    name: str,
+    asset_name: str,
+    vis_params: VisualizeParams,
+) -> QgsRasterLayer:
     """
     Adds an EE image from a catalog.
     """
     image = ee.Image(asset_name).visualize(vis_params)
```

Comment:
```suggestion
    image = ee.Image(asset_name).visualize(**vis_params)
```

I think this may be a bug. @zacdezgeo should we commit this also?